apache/superset
# fix: Return original document title when leaving a dashboard

### SUMMARY
This PR replaces the document title back to the original one when leaving the dashboard. This is to make sure that the dashboard title does not remain in the browser tab while moving to different pages without reloading.

This is not a definitive solution but a quick fix for the problem reported in #16254. As we move toward eliminating page reloads, the need for a title management implementation arises. In fact, the titles are currently only managed server-side, with the Dashboard page being the only exception.

### BEFORE

https://user-images.githubusercontent.com/60598000/129903630-e99f92f6-733b-4566-82a9-b7f9ff09a715.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/129903675-df40022d-e4d8-443a-82cf-8e198e1eb8f1.mp4

### TESTING INSTRUCTIONS
1. Go to the Dashboards' list
2. Open a Dashboard and observe the browser tab title
3. Go back to the Dashboards' list and make sure the browser tab title has changed

### ADDITIONAL INFORMATION


- [x] Has associated issue: Fixes #16254
